<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>561</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLineEdit" name="array_lineEdit">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Введите массив:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QPushButton" name="fib_btn">
       <property name="text">
        <string>Ок</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="fib_lineEdit">
       <property name="placeholderText">
        <string>Числа Фибоначи:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="fib_size_lineEdit">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Введите размер массива чисел Фибоначчи:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="set_lineEdit">
       <property name="placeholderText">
        <string>Уникальные значения:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" rowspan="2">
      <widget class="QPushButton" name="array_btn">
       <property name="text">
        <string>Ок</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
